lms/templates/goal_reminder_banner.html

```diff
@@ -1,59 +1,18 @@
-{% load i18n %}
 {# email client support for style sheets is pretty spotty, so we have to inline all of these styles #}
-{# we're using important below to override inline styles and my understanding is for email clients where media queries do not work, they'll simply see the desktop css on their phone #}
 <style>
```

Comment:
i think gmail might not support style tags outside of head.  
but this PR would still be an incremental improvement while I work on identifying a solution for that

Reply:
You should be able to test this manually locally though before merging though, correct?